features/cucumber.feature

```diff
@@ -138,11 +138,7 @@ Feature: The cucumber command
       And the output should contain "> Using parallel_tests"
 
 
-  Scenario: Using the @solo tag
-
-    The cucumber command runs all features without the @solo tag. If any of the
-    specified .feature files contains '@solo', it boots Cucumber a second time
-    and runs only those of the features tagged with @solo.
+  Scenario: The usage of @solo tags is ignored
```

Comment:
Drop this scenario as well. The `@solo` tag is no longer part of geordi's functionality, so we don't need any tests concerning `@solo` anymore.